src/firefly/js/data/ServerParams.js

```diff
@@ -129,6 +131,11 @@ export const ServerParams = {
         TITLE: 'Title',
         JSON_DEEP: 'jsonDeep',
         ACTION: 'action',
+        PACKAGE_REQUEST: 'packageRequest',
+        TABLE_SEARCH: 'tableSearch',
+        SELECTED_VALUES: 'selectedValues',
+        TABLE_SAVE: 'tableSave',
+        JSON_SEARCH: 'jsonSearch',
```

Comment:
TABLE_SEARCH is a search that returns a TableModel.
JSON_SEARCH returns a JSON string which can be anything.
It is similar to JSON_DATA but uses the new json-ized TableRequest.
It's not used much, and I do plan to consolidate the two.  Backward compatibility was a concern before.  Not so anymore.